Realm/RLMQueryUtil.mm

```diff
@@ -18,17 +18,18 @@
 
 #import "RLMQueryUtil.hpp"
 
-#import "RLMArray_Private.hpp"
+#import "RLMArray.h"
+#import "RLMObjectSchema.h"
 #import "RLMObject_Private.hpp"
-#import "RLMObjectSchema_Private.hpp"
 #import "RLMPredicateUtil.hpp"
-#import "RLMProperty_Private.h"
-#import "RLMSchema_Private.h"
+#import "RLMProperty.h"
+#import "RLMSchema.h"
 #import "RLMUtil.hpp"
 
+#import "object_store.hpp"
 #import "results.hpp"
 
-#include <realm.hpp>
+#include <realm/query_engine.hpp>
```

Comment:
We need query_expression.hpp but that doesn't actually compile without query_engine.hpp.